pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -794,6 +796,23 @@ func (c *Cluster) failoverMon(name string) error {
 	return c.removeMon(name)
 }
 
+func (c *Cluster) stopMonDuringFailover(name string) bool {
+	if !c.spec.Network.IsHost() {
+		return true
+	}
+
+	// If the mon is using host networking, we don't want to stop the mon
+	// since we don't want to schedule a new mon with the same host ip.
+	// With host networking, when failing over the mon, we either require
+	// another eligible node without a mon, or we expect the down mon to come back online.
+	// But if the host networking settings are being enabled, we can allow the failover from a
+	// non-host networking mon.
```

Comment:
What about the vice versa condition? That is, if host-networking is being disable. Will that be supported with current changes?